nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/scheduler/SchedulerServiceImpl.java

```diff
@@ -108,9 +109,13 @@ protected void setupScheduler() throws IOException, SchedulerException {
         // https://jira.nuxeo.com/browse/NXP-7303
         GroupMatcher<JobKey> matcher = GroupMatcher.jobGroupEquals("nuxeo");
         Set<JobKey> jobs = scheduler.getJobKeys(matcher);
-        scheduler.deleteJobs(new ArrayList<JobKey>(jobs));
-        for (Schedule each : registry.getSchedules()) {
-            registerSchedule(each);
+        try {
+            scheduler.deleteJobs(new ArrayList<JobKey>(jobs));
+            for (Schedule each : registry.getSchedules()) {
+                registerSchedule(each);
+            }
+        } catch (LockException cause) {
+            log.warn("scheduler already re-initializing, another cluster node concurrent startup ?", cause);
         }
```

Comment:
The lock exception is raised by the delete, but I want to abort the update in that case. That's why I've kept the loop inside the try block. Also, I've tried to define a warn message according to this. I'm open to any suggest for a better comment.